lib/Resource.js

```diff
@@ -38,7 +38,14 @@ var Resource = function(options) {
   this.search = options.search;
   this.sort = options.sort;
 
+  this.associationOptions = {
```

Comment:
@mbroadst If it is added to the object `options` (L12) then the option `associations: true` will remove the default values. No ?